utils/markdown.js

```diff
@@ -162,12 +163,20 @@ function buildTree(lines) {
 
     // Remaining items respect indent-based nesting.
     const [, indent, content] = /** @type {string[]} */ (line.match('^([ ]*)(.*)'));
+    const [codeLang, title] = (() => {
```

Comment:
- Let's only do this inside if-block at line 173.
- Please extract a named function.